.golangci.yml

```diff
@@ -129,16 +18,72 @@ linters:
     - exhaustruct
     - forcetypeassert
     - funlen
-    - gci
     - gochecknoglobals
     - gocognit
     - goconst
     - ireturn
     - lll
     - mnd
     - nlreturn
+    - noinlineerr
     - perfsprint
-    - tenv
+    - revive
```

Comment:
`tenv` removed by migration script

`revive` added because it produces a single error telling you to rename the `util` package to something more meaningful, which is a bit too opinionated in my view